overlays/python-inconsistent-interpreters.nix

```diff
@@ -0,0 +1,39 @@
+{ builtAttrs
+, packageSet
+, namePositions
+}@attrs:
+
+final: prev:
+let
+  inherit (prev) lib;
+  inherit (import ../lib { inherit lib; }) checkBuildPythonPackageFor;
+
+  checkDerivation = python: drv:
```

Comment:
I am thinking that instead of `python` or `stdenv`, we could just pass `originalDrv` and use `originalDrv.pythonModule` here. This would give us much more information to use and also can be easily fitted into the extracted function from https://github.com/jtojnar/nixpkgs-hammering/pull/17

Reply:
Yes, that’s a very good idea.